app/models/file_node.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 class FileNode < Valkyrie::Model
+  include Valkyrie::Model::AccessControls
```

Comment:
Are you imagining classes that extend `Valkyrie::Model` but do not include the access controls? It feels a little like an expectation that `Model` subclasses will have those four attributes. If this is just a convenience module for a standard set of attributes, I'd consider a `StandardModel` subclass (or some such). That is, unless you expect people to "go shopping" from a list of typical access control schemes (and including the appropriate module -- but again, is that really just convenient inheritance?)